src/ServiceControlInstaller.Engine/LicenseMgmt/LicenseManager.cs

```diff
@@ -40,17 +41,16 @@ public static bool TryImportLicense(string licenseFile, out string errorMessage)
 
             try
             {
-                using (var localMachine = RegistryKey.OpenBaseKey(RegistryHive.LocalMachine, Environment.Is64BitOperatingSystem ? RegistryView.Registry64 : RegistryView.Default))
-                {
-                    new RegistryLicenseStore(localMachine).StoreLicense(licenseText);
-                }
+                new RegistryLicenseStore(Registry.LocalMachine).StoreLicense(licenseText);
             }
             catch (Exception)
             {
                 errorMessage = "Failed to import license into the registry";
                 return false;
             }
 
+            //TODO: import license into new registry location
```

Comment:
when the new licensing package is available (this is why this PR is marked as WIP)